backend/src/composite/split.rs

```diff
@@ -63,7 +63,7 @@ pub(crate) fn machine_witness_columns<F: FieldElement>(
                 panic!("Machine {machine_name} has witness columns of different sizes")
             }
         });
-    let dummy_column_name = format!("{machine_name}.{DUMMY_COLUMN_NAME}");
+    let dummy_column_name = format!("{machine_name}::{DUMMY_COLUMN_NAME}");
```

Comment:
Also a value.

Reply:
But isn't it forming the column namespace?

Reply:
ah right, machine name is a namespace here not a powdr machine name!